plenum/server/node.py

```diff
@@ -1016,14 +1077,15 @@ def _ask_for_ledger_status(self, node_name: str, ledger_id):
                      .format(self, node_name, ledger_id))
 
     def send_ledger_status_to_newly_connected_node(self, node_name):
-        self.sendPoolLedgerStatus(node_name)
+        self.sendLedgerStatus(node_name,
+                              self.ledgerManager.ledger_sync_order[0])
         # Send the domain ledger status only when it has discovered enough
         # peers otherwise very few peers will know that this node is lagging
         # behind and it will not receive sufficient consistency proofs to
         # verify the exact state of the ledger.
-        # if self.mode in (Mode.discovered, Mode.participating):
         if Mode.is_done_discovering(self.mode):
-            self.sendDomainLedgerStatus(node_name)
+            for lid in self.ledgerManager.ledger_sync_order[1:]:
```

Comment:
For all, since the LedgerManager stashes LedgerStatus of all ledger ids it is supposed to sync later so if the LedgerManager has not completed syncing config ledger and it gets status of domain ledger, it will stash it and apply post syncing config ledger

Reply:
Ok